### ISSUE TITLE
Hidden songs of the meta playlist

### ISSUE THREAD

**Issue by lucile-sticky** [ogamiitto/barde#8]
# User
All

# Pages
Top songs

# Problem
Songs are randomly ordered in the list (by rank, and by creation date I suppose).

The playlist contains only 20 songs, so some of them having the same rank are not displayed, depending of the sorting.

**e.G.** *Hard to concentrate* has a 9.5 and is displayed, and some other also have a 9.5 and are not displayed.

# Idea
Display all the songs that have a ```rank > 9``` (or ```> 8```). The playlist will be longer but all the songs with the same rank will be displayed.

**Comment by ogamiitto** [comment]
The song are ordered by higher rank, then latest submit first.
I use 20 as it is the actual "standard" length for the current playlists and I would like to keep this length for now.
To understand better your idea, you do not want a rank to be broken in the top list ?

**Comment by lucile-sticky** [comment]
Yeah, that's the idea, because it's not "fair" that some songs having the same rank than the lasts in the list are not displayed.
